lib/src/json_schema/json_schema.dart

```diff
@@ -131,7 +131,7 @@ class JsonSchema {
           .future;
 
       // Boolean schemas are only supported in draft 6 and later.
-    } else if (data is bool && [SchemaVersion.draft6, SchemaVersion.draft7].contains(version)) {
+    } else if (data is bool && version != SchemaVersion.draft4) {
```

Comment:
How would you feel about using `version.compareTo(SchemaVersion.draft6) >= 0` for this?

Reply:
Talked in person. We'll have some followup work to clean up how we compare schema versions.